crates/next-core/js/src/overlay/internal/container/Errors.tsx

```diff
@@ -154,8 +154,10 @@ export function Errors({ issues, errors }: ErrorsProps) {
   );
 
   const hasIssues = issues.length !== 0;
-  const hasIssueWithError = issues.some((issue) =>
-    ["bug", "fatal", "error"].includes(issue.severity)
+  const hasIssueWithError = issues.some(
+    (issue) =>
+      ["bug", "fatal", "error"].includes(issue.severity) &&
+      !(issue.severity === "error" && issue.category === "resolve")
```

Comment:
Could you add a short comment explaining why we don't consider resolve issues as errors?

Reply:
I think we shouldn't use the category for that decision. This should be encoded in the `severity` only. So:
* reduce resolve errors to `warning`, or
* allow to hide `error`s in general

I feel like resolve errors are a bit weak as `warnings`. So I would prefer keeping them errors.

Maybe we need a new severity type that is between `fatal` and `error`.

For reference the descriptions of the error types:

```
            IssueSeverity::Fatal => "unrecoverable problem",
            IssueSeverity::Error => "problem that cause a broken result",
            IssueSeverity::Warning => "problem should be addressed in short term",
```

There would be space for a `problem that cause a partial broken result`...

Reply:
I accidentally unmarked this as a draft, just wanted to keep this around because I needed it at some point